extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -19,7 +19,8 @@ export function registerTableDesignerCommands(appContext: AppContext) {
 			database: context.connectionProfile.databaseName,
 			isNewTable: true,
 			id: generateUuid(),
-			connectionString: connectionString
+			connectionString: connectionString,
+			accessToken: context.connectionProfile.options.azureAccountToken
```

Comment:
Is there a reason you wouldn't make a connection and then use that connection for the designer instead of passing in these properties yourself? Duplicating the connection properties here doesn't seem great

Reply:
I've created an issue to track this for April release: https://github.com/microsoft/azuredatastudio/issues/18250